mava/utils/make_env.py

```diff
@@ -59,31 +60,30 @@
 # Define a different registry for Matrax since it has no generator.
 _matrax_registry = {"Matrax": MatraxWrapper}
 
-_jaxmarl_wrappers = {"Smax": SmaxWrapper, "MaBrax": MabraxWrapper}
+_jaxmarl_wrappers: Dict[str, Type[JaxMarlWrapper]] = {"Smax": SmaxWrapper, "MaBrax": MabraxWrapper}
```

Comment:
Is the typing here was required for jaxmarl? Can't we do similar thing to jumanji's wrappers?